tools/verify_headers.py

```diff
@@ -82,9 +86,9 @@ def main():
     res = pool.map(validate_header, files)
     failed_files = [x for x in res if x[1] is False]
     if len(failed_files) > 0:
-        for i in failed_files:
-            sys.stderr.write("%s failed header check because:\n\n" % i[0])
-            sys.stderr.write("%s\n\n" % i[2])
+        for failed_file in failed_files:
+            sys.stderr.write("%s failed header check because:\n" % failed_file[0])
```

Comment:
You didn't like the blank line?

Reply:
I do not.. specially when you have more than 5 of them :)